examples/gallery/images/grdgradient_shading.py

```diff
@@ -0,0 +1,61 @@
+"""
+Calculating grid gradient, azimuth, and normalize
+--------------------------------------
+The :func:`pygmt.grdgradient` function calculates the gradient of a grid file.
+As input :func:`pygmt.grdgradient` gets a :class:`xarray.DataArray` object or
+a path string to a grid file, calculates the respective gradient and returns
+it as an :class:`xarray.DataArray` object. The ``azimuth`` parameter in order
+to set the illumination light source direction. The ``normalize`` parameter
+enhances the three-dimensional sense of the topography. It calculates the
+azimuthal gradient of each point along a certain azimuth angle, then adjust
+the brightness value of the color according to the positive/negative of the
+azimuthal gradient and the amplitude of each point.
+"""
+
+import pygmt
+
+# Define region of interest around Caucasus
+region = [35, 50, 35, 45]
+
+# Load sample grid (1 arc-minutes global relief) in target area
+grid = pygmt.datasets.load_earth_relief(resolution="01m", region=region)
```

Comment:
The example takes 16 s to run on the CI. Perhaps use a slightly lower-resolution dataset (e.g., `03m` or `04m`) to speed up?

Reply:
Your consideration is greatly appreciated, thank you.